r/R/compute.R

```diff
@@ -241,13 +241,11 @@ all.ArrowDatum <- function(..., na.rm = FALSE){
 #' `boolean`-type `Array` of the same length as `x` with values indicating
 #' per element of `x` it it is present in `table`.
 #' @export
-match_arrow <- function(x, table, ...) UseMethod("match_arrow")
-
-#' @export
-match_arrow.default <- function(x, table, ...) match(x, table, ...)
```

Comment:
This is probably over-board, but we are changing the behavior here, might we want a warning here (maybe temporary? maybe only once per session) in the case where both `x` and `table` are not Arrow objects and mention that this behavior is different? 

Maybe an entry in news describing this would be sufficient. The use of `match_arrow()` to dispatch `base::match()` is certainly an off-label use

Reply:
@jonkeane That makes sense, it is a breaking change after all.  Just committed an update - let me know if I should rephrase it.

Reply:
That looks good